src/content/docs/iast/troubleshooting/unable-to-find-known-vulnerabilities.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Unable to find known vulnerabilities 
+tags:
+    - IAST
+    - New Relic IAST
+    - Troubleshooting
+metaDescription: Unable to find known vulnerabilities 
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+I tested an application with known vulnerabilities, but IAST couldn't detect these.
+
+
+## Solution
+<Callout variant="important">
+IAST starts testing when it detects some traffic, so check if your application works out or has any traffic directed at it. Perform the app's UI or API endpoints. 
+Also, IAST requires restarting the application after it's enabled.
+</Callout>
+
+Ensure that you are using a [suppoted application framework](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/) and the [latest instrumentation package](https://github.com/newrelic/csec-go-agent#instrumentation-packages) for your application. 
+
+
+## Cause
+
+Following could be the reasons if IAST doesn't detect known vulnerabilities:
```

Comment:
I'd prefer to rewrite this as something like:

"These are possible reasons IAST isn't detecting any known vulnerabilities"